src/Workspaces/CoreTestUtilities/OptionsCollection.cs

```diff
@@ -4,12 +4,15 @@
 
 #nullable enable
 
-using System;
 using System.Collections;
 using System.Collections.Generic;
 using Microsoft.CodeAnalysis.CodeStyle;
 using Microsoft.CodeAnalysis.Options;
+
+#if !NETCOREAPP
+using System;
 using Roslyn.Utilities;
```

Comment:
Why this #if is needed?

Reply:
The using directives are only used in one framework.

Reply:
Minor curiousity: what about `Roslyn.Utilities` gets used in .NET Core here? There aren't any particular code changes in the PR that hook into new APIs.

Reply:
@jaredpar None of them do, hence the exclusion for that build 😄

Reply:
🤦 

This is what I get for commenting before finishing my coffee